docs/resources/file.md.erb

```diff
@@ -87,6 +87,22 @@ The `mode` property tests if the mode assigned to the file matches the specified
 
     its('mode') { should cmp '0644' }
 
+You must write the numeric mode specification starting with zero, to tell InSpec
+the value is [octal](https://en.wikipedia.org/wiki/Leading_zero#0_as_a_prefix).
+
+For example, write:
+
+    { should cmp '0644' }
+ 
+If you write it as
```

Comment:
Without the zero prefix for the octal value, the InSpec will interpret the it as the _decimal_ value 644, which is octal 1024 or`-----w-r-T`, and any test for a file that is `-rw-r--r--` will fail.